src/components/InputField/InputField.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {TextInputProps, Text, View} from 'react-native';
+import {TextInput} from 'react-native-paper';
+
+interface Props extends TextInputProps {
+  error?: string;
+  touched?: boolean;
+  label: string;
+  value: string;
+  mode: 'flat' | 'outlined' | undefined;
```

Comment:
also we can make it simpler:
```suggestion
  mode?: 'flat' | 'outlined';
```